Common/Tests/Utilities.UI/UI/ReplWindowProxy.cs

```diff
@@ -349,6 +411,22 @@ string executionMode
                 .Select(l => l.TrimEnd('\r', '\n', ' '))
                 .ToList();
 
+#if DEV14_OR_LATER
+            dynamic eval = _window.Evaluator;
+            var primary = eval.PrimaryPrompt as string ?? ">>>";
+#else
```

Comment:
This file, ReplWindowProxy.cs, with MockReplWindow.cs and InteractiveWindow.cs are physically in the common folder, but compiled as part of TestUtilities.Python.

Even before my changes, ReplWindowProxy.cs and InteractiveWindow.cs have Python specific stuff hard coded in them.

I'll try to move all 3 files to the TestUtilities.Python folder on disk. change namespace to Python and merge the base and derived classes.